src/material/tooltip/tooltip.ts

```diff
@@ -641,42 +651,49 @@ export class TooltipComponent implements OnDestroy {
   /** The timeout ID of any current timer set to hide the tooltip */
   _hideTimeoutId: number | null;
 
-  /** Property watched by the animation framework to show or hide the tooltip */
-  _visibility: TooltipVisibility = 'initial';
-
   /** Whether interactions on the page should close the tooltip */
-  private _closeOnInteraction: boolean = false;
+  private _closeOnInteraction = false;
+
+  /** Whether the tooltip is currently visible. */
+  private _isVisible = false;
 
   /** Subject for notifying that the tooltip has been hidden from the view */
-  private readonly _onHide: Subject<any> = new Subject();
+  private readonly _onHide = new Subject<void>();
 
   /** Stream that emits whether the user has a handset-sized display.  */
   _isHandset: Observable<BreakpointState> = this._breakpointObserver.observe(Breakpoints.Handset);
 
+  /** Reference to the internal tooltip element. */
+  @ViewChild('tooltip', {
+    // Use a static query here since we interact directly with
+    // the DOM which can happen before `ngAfterViewInit`.
+    static: true
+  }) _tooltip: ElementRef<HTMLElement>;
+
   constructor(
     private _changeDetectorRef: ChangeDetectorRef,
-    private _breakpointObserver: BreakpointObserver) {}
+    private _breakpointObserver: BreakpointObserver,
+    @Optional() @Inject(ANIMATION_MODULE_TYPE) public _animationMode?: string) {}
 
   /**
    * Shows the tooltip with an animation originating from the provided origin
    * @param delay Amount of milliseconds to the delay showing the tooltip.
    */
-  show(delay: number): void {
+  show(delay: number, isUserInteraction?: boolean): void {
```

Comment:
Technically this is a breaking change since we export `TooltipComponent` in the public API (probably unintentionally)

Reply:
I don't think it's breaking since the new parameter is optional.